### DISCUSSION TITLE
I had an error installing

### DISCUSSION THREAD

**Discussion by phamthanhnghia** [balajidharma/laravel-vue-admin-panel#43] [Q&A]
I tried to install source , even though I granted the .env variable but still got the error. I ran the command as instructed in the README, I used window 11 and ran it on wsl terminal. Appreciate your support. 

```
WWWUSER=root
WWWGROUP=www-date
```
Error :
```
 > [laravel.test  6/11] RUN groupadd --force -g  sail:
0.481 groupadd: invalid group ID 'sail'
failed to solve: process "/bin/sh -c groupadd --force -g $WWWGROUP sail" did not complete successfully: exit code: 3
```

**Comment by balajidharma** [comment]
@phamthanhnghia This error is related to the sail build... Check the Laracasts discussion https://laracasts.com/discuss/channels/laravel/groupadd-invalid-group-id-sail**
